ruby/ql/lib/codeql/ruby/typetracking/ApiGraphShared.qll

```diff
@@ -185,7 +185,7 @@ module ApiGraphShared<ApiGraphSharedSig S> {
   bindingset[sink]
   pragma[inline_late]
   Node getAValueReachingSinkInline(ApiNode sink) {
-    result = asSinkInline(getAnEpsilonSuccessorInline(sink))
+    backwardStartNode(result) = getAnEpsilonSuccessorInline(sink)
```

Comment:
I forgot to ask, what does this change do?

Reply:
You'd have to ask Asger 😅 - this fix is from https://github.com/alexrford/codeql/pull/33/commits/fe6f4fad98b8941b910390ce456425cc9e239c1a but I don't have a full description of what it fixes.

I think that the previous definition was too restricted in which nodes it found, as in `asSinkInline` has more limited tracking than `backwardStartNode`. Without this change we lose a lot of legitimate type defs.